root/usr/bin/nm-configure

```diff
@@ -48,18 +56,19 @@ method=ignore
 [vlan]
 id=$vlanid
 EONS
-  correct_owner_perms $script
+  deploy_config $TMP_CFG $script
 elif [[ "$1" == "primary" ]]; then
   vlanid=${3:-$KCL_FDI_VLAN_PRIMARY}
   [[ "$vlanid" != "" ]] && ctype="vlan"
   script=/etc/NetworkManager/system-connections/primary
-  cat > $script <<EONS
+  cat > $TMP_CFG <<EONS
 [connection]
 id=primary
 uuid=$uuid
 type=$ctype
 autoconnect=true
 autoconnect-priority=1
+may-fail=false
```

Comment:
Not much relevant, I was just browsing docs trying to find something and found this option.